### PR TITLE
Add WeChat to referral docs

### PR DESCRIPTION
**Pull request by spasiu**
Add ref info for WeChat to `conversation:start` and `:referral` docs.

### CODE REVIEW

File: slate/source/includes/rest/core/_conversations.md [hunk 1]
```diff
@@ -1221,7 +1221,7 @@ Upload an attachment to Smooch to use in future messages. Files are uploaded usi
 | **access**<br/><span class='req'>required</span>    | The access level for the attachment. Currently the only available access level is `public` |
 
 <aside class="notice">
-The maximum size allowed per file is 10MB. Exceeding this size will result in a [`413 error`](#errors). 
+The maximum size allowed per file is 10MB. Exceeding this size will result in a [`413 error`](#errors).
```
Comment (spasiu) [inline slate/source/includes/rest/core/_conversations.md]: I guess I nuked some whitespace here...

File: slate/source/includes/rest/core/webhooks/_conversation_events.md [hunk 2]
```diff
@@ -66,6 +66,11 @@ The payload for when a user reads a conversation.
 
 The payload for when a user opts in to start receiving messages. `conversation:start` is only available on a sub set of channels. [Channel capabilities](https://docs.smooch.io/guide/channel-capabilities/) lists which channel currently supports it. Also, note that `conversation:start` won't be triggered when a user is linking a second channel via the Web Messenger.
 
+The `conversation:start` event can contain a referral property in the following circumstances:
+
+- When a [referral parameter](https://developers.facebook.com/docs/messenger-platform/referral-params) is included with a link to Facebook Messenger.
+- When - A WeChat [subscribe event](http://admin.wechat.com/wiki/index.php?title=Event-based_Messages#Scanning_Parametric_QR_Code_Event) event is triggered by a new follower and the QR code contains an embedded referral property.
```
Comment (mspensieri) [inline slate/source/includes/rest/core/webhooks/_conversation_events.md]: there's an extra `-` after When here.

Also, `subscribe event event`

File: slate/source/includes/rest/core/_conversations.md [hunk 3]
```diff
@@ -1424,12 +1424,12 @@ This table represents the fields you can expect to receive nested inside postbac
...
 
 | Field       | Description                                                                  |
 |-------------|------------------------------------------------------------------------------|
-| **code**    | The referral's identifier                                                    |
-| **details** | Nested object containing `source`, `type` and `adId`                         |
-| **source**  | The source of the referral. Ex: `MESSENGER_CODE`, `ADS` etc...               |
-| **type**    | The type of referral, usually `OPEN-THREAD`                                  |
-| **adId**    | If the referral came from an ad, this field will be present with the ad's Id |
+| **code**    | The referral's identifier. Available in referrals from WeChat and Messenger |
+| **details** | Nested object containing `source`, `type` and `adId`. Only available on Messenger |
```
Comment (mspensieri) [inline slate/source/includes/rest/core/_conversations.md]: If this is an object, then `source`, `type`, and `adId` should instead be in a separate table called "Referral Details Schema"

File: slate/source/includes/rest/core/webhooks/_conversation_events.md [hunk 4]
```diff
@@ -102,8 +107,9 @@ The payload for when a user opts in to start receiving messages. `conversation:s
 | **appUser**  | A [truncated appUser](#truncated-app-user-schema) schema object |
 | **referral** | Referral information. See [referral schema](#referral-schema) for details. |
 
-The payload for when a user is referred to a conversation. Currently, Messenger is the only channel that supports this.
+The payload for when a user is referred to a conversation.
 This payload will be sent when a user performs the following actions:
 
 - Scanning a [Messenger code](https://developers.facebook.com/docs/messenger-platform/messenger-code)
 - Clicking a [conversion ad](https://developers.facebook.com/docs/messenger-platform/guides/ads) on Facebook
+- A WeChat [scan event](http://admin.wechat.com/wiki/index.php?title=Event-based_Messages#Scanning_Parametric_QR_Code_Event) (when the user is a follower).
```
Comment (jpjoyal) [inline slate/source/includes/rest/core/webhooks/_conversation_events.md]: Nit: Keep the same style eg: "Scanning a [WeChat QR code](link) after following a business.

**Pull request merged**